src/DefaultEmissionManager.sol

```diff
@@ -27,6 +27,9 @@ contract DefaultEmissionManager is Ownable2StepUpgradeable, IDefaultEmissionMana
     IPolygonEcosystemToken public token;
     uint256 public startTimestamp;
 
+    // NEW STORAGE 1.2.0
+    uint256 public START_SUPPLY_1_2_0;
```

Comment:
New state variables are introduced without removing 1 bytes32 slot from the reserved gap. After a quick look this time it seems not a real storage issue because there are no more inherits but it is always better to follow all best practices possible. Anyway we strongly recommend to use an `immutable` for that new variable instead of a normal state variable, so in that case, the gap could remain the same.

Furthermore, let's keep lower case for state variables and upper case for constants and immutables.

Reply:
How could i make it immutable? I need it in the reinitialize func, so that I can the proxy storage.
good catch on the storage gap slot though, will remove one to be safe.